data/_models/shared/Catalogues.csv

```diff
@@ -4,4 +4,6 @@ Catalogues,,name,,1,TRUE,,,,,,http://purl.org/dc/terms/title,The name given to t
 Catalogues,,network,ref,1,TRUE,,Resources,,,,,Network or consortium that publishes this catalogue,DataCatalogueFlat,,
 Catalogues,,type,ontology,,TRUE,CatalogueOntologies,CatalogueTypes,,,,,Type of catalogue,DataCatalogueFlat,,
 Catalogues,,description,text,,TRUE,,,,,,http://purl.org/dc/terms/description,Description of this catalogue,DataCatalogueFlat,,network?.description
-Catalogues,,publisher,,,,,,,,,http://purl.org/dc/terms/publisher,The organisation responsible for making the catalogue available,DataCatalogueFlat,,
+Catalogues,,publisher,ref_array,,TRUE,,Agent,,,,http://purl.org/dc/terms/publisher,The organisation responsible for making the catalogue available,DataCatalogueFlat,,
+Catalogues,,license,hyperlink,,TRUE,,,,,,http://purl.org/dc/terms/license,The reference to the usage license of the catalog.,DataCatalogueFlat,,
+Catalogues,,conformsTo,hyperlink,,TRUE,,,,,,http://purl.org/dc/terms/conformsTo,The specification of the catalog metadata schema,DataCatalogueFlat,,
```

Comment:
offtopic: what would be realistic values of this?

Reply:
I suppose adding a license to published data is good practice either way, not so sure about conformsTo though

Reply:
good point, let's do a name check once this is merged :)